config.py

```diff
@@ -475,6 +475,33 @@
             '.*[0-9]+.*',
         ],
     ),
+    
+    fi = dict(
+        lang_name = 'Suomi',
+        lang_dir = 'ltr',
+        database = 'fiwiki_p',
+        wikipedia_domain = 'fi.wikipedia.org',
+
+        beginners_link = 'https://fi.wikipedia.org/wiki/Wikipedia:Merkitse_l%C3%A4hteet',
```

Comment:
How about using https://fi.wikipedia.org/wiki/Ohje:L%C3%A4hteen_lis%C3%A4%C3%A4minen_visuaalisella_muokkaimella/1? That seems more beginner-friendly and equivalent to what we do for the English version.